package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -1,25 +1,87 @@
 #!/usr/bin/lua
+-- TODO: replace REJECT by DROP. only use REJECT for debugging
 
 local site = require 'gluon.site_config'
 local uci = require('simple-uci').cursor()
 
+uci:section('firewall', 'zone', 'wan', {
+	name = 'wan',
+	network = { 'wan','wan6',},
+	input = 'REJECT',
+	output = 'ACCEPT',
+	forward = 'REJECT',
+	mtu_fix = '1',
+	conntrack = '1',
+	log = '1',
+}
+)
+-- the client zone is set up by gluon-client-bridge
+--
+uci:section('firewall', 'zone', 'mesh',
+{
+	name = 'mesh',
+	input = 'REJECT',
+	output = 'ACCEPT',
+	forward = 'REJECT',
+	conntrack = '1',
+	log = '1',
+}
+)
 
-local function reject_input_on_wan(zone)
-	if zone.name == 'wan' then
-		uci:set('firewall', zone['.name'], 'input', 'REJECT')
-		uci:set('firewall', zone['.name'], 'conntrack', true)
-	end
+-- allow inbound ssh from anywhere
+for _, zone in ipairs({ 'wan', 'client', 'mesh' })
+do
+	uci:section('firewall', 'rule', zone .. '_ssh',
+	{
+		name =  zone .. '_ssh',
+		src = zone,
+		dest_port = '22',
+		proto = 'tcp',
+		target = 'ACCEPT',
+	}
+	)
```

Comment:
The indentation of all these uci:section blocks doesn't match the general coding style commonly used in Gluon.